src/modules/Tickets/TicketController.ts

```diff
@@ -7,46 +7,68 @@ import {
   Post,
   Put,
   Delete,
+  JsonController,
 } from "routing-controllers";
 
 import { TicketServices } from "./TicketServices";
+import { Container } from "typedi";
+import { ITicket } from "@app/data/abstraction/entities/ITicket";
+import _ from "lodash";
+import { NotAllow } from "@app/decorator/notAllow";
 
 @Controller()
 export class TicketController {
+  serviceInstance = Container.get(TicketServices);
+
   @Get("/")
-  response() {
-    return { msg: "Welcome" };
+  async welcomeMessage() {
+    try {
+      return "Welcome";
+    } catch (error) {
+      return;
+    }
   }
+
   @Get("/tickets")
-  getAll() {
-    const services = new TicketServices();
-    return services.getAll();
+  async fetchAllTickets() {
+    try {
+      const services = await this.serviceInstance.fetchAllTickets();
+      return services;
+    } catch (error) {
+      return error;
+    }
   }
```

Comment:
try catch in this case doesn't do any thing
```suggestion
    const services = await this.serviceInstance.fetchAllTickets();
  }
```